cmd/dbmigrate/cmd/dbmigrate.go

```diff
@@ -73,6 +73,10 @@ Migration process:
    and by default will be in the {home} directoyr.
 3. Move the staging data directory into place as the current data directory.
 4. Update the config file to reflect the new database backend type.
+
+Source Database Type:
+The source database type is read from the config.toml within the home directory. If this does not match the database types, then the conversion
+process will not work properly.
```

Comment:
To address this, I'd rather add a `--source-db-backend` flag to the command that could be provided to the migrator. If it's provided, use that for all dbs, otherwise, auto-detect for each.

I didn't add this originally because there's no guarantee that all dbs in the directory have the same backend. Also because the `application` and `snapshots` db backend types were originally defined using a set-at-compile-time `DBBackend` variable; the`dbmigrator` had no way of knowing what that value was.